rules/LineTest.js

```diff
@@ -1,17 +1,22 @@
+/**
+ * This is used by ScriptMatchTest. 
+ *
+ * TODO: Could we use Line.js instead?
+ */
```

Comment:
This is still needed by one of the tests. Would like to replace it with `Line.js`, if possible.